tests/shell/030_get_set_input.sh

```diff
@@ -0,0 +1,99 @@
+set -e
+set -x
+cd tests/
+mkdir -p ./tmp/
+cd ./tmp/
+touch cfbs.json && rm cfbs.json
+rm -rf .git
+rm -rf create-single-file
+
+echo '{
+  "build": [
+    {
+      "name": "create-single-file",
+      "input": [
+        {
+          "type": "string",
+          "variable": "filename",
+          "label": "Filename",
+          "question": "What file should this module create?"
+        }
+      ]
+    }
+  ]
+}' > cfbs.json
+
+# Igor asks for input
+cfbs get-input create-single-file > actual.output
+echo '[
+  {
+    "type": "string",
+    "variable": "filename",
+    "label": "Filename",
+    "question": "What file should this module create?"
+  }
+]' > expected.output
+diff actual.output expected.output
+
+# Igor adds response with php magic
+echo '[
+  {
+    "type": "string",
+    "variable": "filename",
+    "label": "Filename",
+    "question": "What file should this module create?",
+    "response": "/tmp/test-1.txt"
+  }
+]' | cfbs set-input create-single-file
+
+# Igor asks for input again
+cfbs get-input create-single-file > actual.output
+echo '[
+  {
+    "type": "string",
+    "variable": "filename",
+    "label": "Filename",
+    "question": "What file should this module create?",
+    "response": "/tmp/test-1.txt"
+  }
+]' > expected.output
+diff actual.output expected.output
+
+# Igor changes the response to something else
+echo '[
+  {
+    "type": "string",
+    "variable": "filename",
+    "label": "Filename",
+    "question": "What file should this module create?",
+    "response": "/tmp/test-2.txt"
+  }
+]' | cfbs set-input create-single-file
+
+# Igor asks for input again
+cfbs get-input create-single-file actual.output
+echo '[
+  {
+    "type": "string",
+    "variable": "filename",
+    "label": "Filename",
+    "question": "What file should this module create?",
+    "response": "/tmp/test-2.txt"
+  }
+]' > expected.output
+diff actual.output expected.output
+
+# Igor changes the wrong field
+echo '[
+  {
+    "type": "string",
+    "variable": "bogus",
+    "label": "Filename",
+    "question": "What file should this module create?",
+    "response": "/tmp/test-2.txt"
+  }
+]' > igors-input.json
+
+! cfbs set-input create-single-file igors-input.json
```

Comment:
Could you add another test like this where one of the keys is changed, like `type` is changed to something else?

Reply:
Sure thing! That a key is changed instead of value?

Reply:
yep, thanks.